test/configurationTests.ts

```diff
@@ -151,7 +152,9 @@ describe("Configuration", () => {
             rawConfig = {
                 jsRules: true,
                 rules: {
-                    eofline: true,
+                    // valid rule for JS, disabled (should be copied over)
+                    eofline: false,
```

Comment:
I tweaked this existing test case to cover the new behavior of keeping disabled rules